planning/behavior_path_planner/include/behavior_path_planner/scene_module/side_shift/side_shift_module.hpp

```diff
@@ -46,6 +50,27 @@ struct SideShiftParameters
   double drivable_area_height;
 };
 
+struct SideShiftRequestTimer
+{
+  time_point<high_resolution_clock> last_request_clock = high_resolution_clock::now();
+
+  bool override_request = true;
+
+  bool isRequestAllowed(const double & change_time)
```

Comment:
What is the unit? msec?

min_request_time_msec might be better.

Reply:
the time value should be in second, i.e: 0.5 second or 1.0 second, so that it sync with `time_to_start_shifting` from `side_shift.param.yaml`